runtime/hydradx/src/xcm.rs

```diff
@@ -217,7 +242,7 @@ parameter_type_with_key! {
 	pub ParachainMinFee: |location: Location| -> Option<u128> {
 		#[allow(clippy::match_ref_pats)] // false positive
 		match (location.parents, location.first_interior()) {
-			(1, Some(Parachain(ASSET_HUB_PARA_ID))) => Some(50_000_000),
+			(1, Some(Parachain(ASSET_HUB_PARA_ID))) => Some(150_000_000),
```

Comment:
why this changed? are you sure it doesnt break xcm ui?

Reply:
This is from a [PR](https://github.com/galacticcouncil/hydration-node/pull/911) we merged into this one. @dmoka